### DISCUSSION TITLE
Admonitions in spec definition

### DISCUSSION THREAD

**Discussion by dmitriyrotaenko** [rohit-gohri/redocusaurus#364] [Ideas]
Hi, we're integrating Redocusaurus for our documentation. Really enjoy the project so far. 

My question is whether Redocuaurus can somehow parse [Docusaurus admonitions](https://docusaurus.io/docs/markdown-features/admonitions) within the spec definition?

Here's a custom 'admonition' that I've created. Basically, a `div` with some styles.
<img width="948" alt="Screenshot 2024-05-22 at 17 38 37" src="https://github.com/rohit-gohri/redocusaurus/assets/52621117/bc43882a-94b8-4664-a81f-27ca8fb39c3d">

And the spec definition looks like the following:

```yaml
openapi: 3.0.1
info:
  title: "Public API"
  version: "1.0"
  description: |
    Gibberish gobbledygook
    
    # Div with custom styles
    <div class="tip">
      Useful tip
    <div/>

paths:
  /api/chat_public_keys:
```

Is it possible to to use Docusaurus admonitions in spec like so:

```yaml
openapi: 3.0.1
info:
  title: "Public API"
  version: "1.0"
  description: |
    Gibberish gobbledygook
    
    # Admonition
   :::tip
   Very important tip.
   :::

paths:
  /api/chat_public_keys:
```
And if no, maybe there're some directions I can dig into? Any help and ideas appreciated 🙌

**Comment by rohit-gohri** [comment]
Sounds like a cool idea but not sure how this will work as the open api rendering is done by redoc and is out of my control
